backend/app/api/v1/lantern_api.py

```
@@ -125,7 +126,13 @@ async def create_lanterns(
     name: str = Form(..., min_length=1, max_length=50, description="사용자 이름 (1~50자)"),
     images: List[UploadFile] = File(..., description="이미지 파일 (jpg, jpeg, png, webp, 각 5MB 이하, 총 3장)"),
     is_public: bool = Form(True, description="랜턴을 공개할지 여부"),
+    date: str = Form(..., description="날짜 (yyyy-mm-dd 형식)"),
 ):
+    try:
+        datetime.strptime(date, "%Y-%m-%d")
+    except ValueError:
+        raise InvalidDateFormatError(date)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)